.cirrus.yml

```diff
@@ -62,6 +62,14 @@ task:
       container:
```

Comment:
We should just remove line 35 on this file (which does the same)

Reply:
I'm confused. Line 35 says `only_if: $CIRRUS_BRANCH == 'dev'` and this setups up CPU and memory :confused:

Reply:
I meant in general, removing $CIRRUS_BRANCH == 'dev' will get AOT build tests on pre-submits.

Reply:
Ah, I see. I don't think it's the same. That will actually attempt to build the APK files, and therefore requires the Android SDK with Gradle installed. This is just runs Dart AOT compiler to check that it can digest the Dart code we're throwing at it. It's a pretty quick step meant to be a smoke-test.

Reply:
Also, it seems to only build the gallery.

Reply:
Ah I see I see